src/content/developers/docs/standards/tokens/erc-20/index.md

```diff
@@ -153,4 +153,4 @@ print("Addr Balance:", addr_balance)
 - [EIP-20: ERC-20 Token Standard](https://eips.ethereum.org/EIPS/eip-20)
 - [OpenZeppelin - Tokens](https://docs.openzeppelin.com/contracts/3.x/tokens#ERC20)
 - [OpenZeppelin - ERC-20 Implementation](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol)
-- [ConsenSys - ERC-20 Implementation](https://github.com/ConsenSys/Tokens/blob/master/contracts/eip20/EIP20.sol)
+- [ConsenSys - ERC-20 Implementation](https://github.com/ConsenSys/Tokens/blob/master/contracts/eip20/EIP20.sol) (deprecated)
```

Comment:
I'm definitely in favour of just removing it.

1. The page is primarily solving for users who want to know what ERC-20 is or want to know how to implement it. I don't think keeping this in solves for either user group.
2. Defaulting to remove deprecated/unmaintained software makes it easy to make the change across all translated languages on the site  (the alternative being that we queue up translations for adding a deprecated label across ~40 languages every time something is out of date)

```suggestion
```